library/Booster/JsonRpc.hs

```diff
@@ -509,4 +516,11 @@ mkLogRewriteTrace
                                     , origin = Booster
                                     , result = Failure{reason = "Internal error: " <> err, _ruleId = Nothing}
                                     }
+                            ApplyEquations.SideConditionsFalse{} ->
+                                Simplification -- FIXME
```

Comment:
What this FIXME refers to?

Reply:
Yeah, maybe acceptable that the field `originalTerm` is not provided here.
Ah... and now I also remember I was going to add simplification traces to the error constructor...